bin/pact-cli.ts

```diff
@@ -16,23 +16,34 @@ cli
 	.option("-h, --host <hostname>", "Host on which to bind the service. Default is localhost.")
 	.option("-l, --log <file>", "File to which to log output to.")
 	.option("-s, --ssl <boolean>", "Use a self-signed SSL cert to run the service over HTTPS. Default is false (HTTP).", cli.BOOL)
-	.option("-o, --cors <boolean>", "Support browser security in tests by responding to OPTIONS requests and adding CORS headers to mocked responses. Default is false.", cli.BOOL)
+	.option("-o, --cors <boolean>", `Support browser security in tests by responding to OPTIONS requests and adding CORS headers
```

Comment:
Did your IDE wrap all of these strings?  You do know this is going to break many of them at runtime?

Reply:
I wrapped them by hand on a logical break point (to address the 160 chars limit).

What do you mean breaks at runtime? I tested by running the cli locally, e.g. here is an output for mock:

```sh
matt ~/development/public/pact-node λ node ./bin/pact-cli.js mock  --help

   pact 6.12.2 - Pact CLI tool

   USAGE

     pact mock

   OPTIONS

     -p, --port <n>                      Port on which to run the service. Default is random.                                                                                    optional
     -h, --host <hostname>               Host on which to bind the service. Default is localhost.                                                                                optional
     -l, --log <file>                    File to which to log output to.                                                                                                         optional
     -s, --ssl <boolean>                 Use a self-signed SSL cert to run the service over HTTPS. Default is false (HTTP).                                                      optional
     -o, --cors <boolean>                Support browser security in tests by responding to OPTIONS requests and adding CORS headers to mocked responses. Default is false.      optional
     -d, --pact-dir <directory>          Directory to which the pacts will be written. Default is cwd.                                                                           optional
     -i, --pact-version <n>              The Pact specification version to use when writing the Pact files. Default is 1.                                                        optional
     -w, --pact-file-write-mode <m>      Controls how pact files are written to disk. One of 'overwrite', 'update', 'merge'                                                      optional
     --consumer <consumerName>           Specify consumer name for written Pact files.                                                                                           optional
     --provider <providerName>           Specify provider name for written Pact files.                                                                                           optional
     --monkeypatch <file>                Absolute path to a Ruby file that will monkeypatch the underlying Pact mock.                                                            optional

   GLOBAL OPTIONS

     -h, --help         Display help
     -V, --version      Display version
     --no-color         Disable colors
     --quiet            Quiet mode - only displays warn and error messages
     -v, --verbose      Verbose mode - will also output debug messages
```

Looks ok?

Reply:
Ah, it's because it's using ` and not ".  The latter would make it break since string cannot be multiline without having a + between them.

Reply:
Hmm, no, you're right. This is fixed now.

Reply:
... for posterity, the `` ` `` works, but it actually leaves the space in (so formatting is shit). Fixed with `"` .